assets/js/blocks/reviews-by-product/edit.js

```diff
@@ -287,7 +290,7 @@ const ReviewsByProductEditor = ( { attributes, debouncedSpeak, error, getProduct
 					</Placeholder>
 				) : (
 					<div className={ classes }>
-						<Block attributes={ attributes } isPreview />
+						<EditorBlock attributes={ attributes } />
```

Comment:
Something about this change doesn't sit right with me. EditorBlock/FrontendBlock seem like they could be the same - from what I see, Editor block just needs some things like orderby hardcoded. That could be passed in via props to a single Block, and if we need to disable interaction it can be wrapped in a `Disabled` component. Is having them split like this really necessary?

Reply:
All the presentational part is still shared between EditorBlock and FrontendBlock via components. But besides the presentation, internally both components work quite differently. Only in the frontend we need to take care of user interaction while only in the editor we need to take care of attributes updating, for example. This change also allow us to use `debounce` in the editor without adding that dependency in the frontend.

If in the future we create a `withReviews()` HOC, those components will be even slimmer so I think it's worth having them separated.

Does that make sense?

Reply:
> If in the future we create a withReviews() HOC, those components will be even slimmer so I think it's worth having them separated.

I think this is key, to me this seems to be the remaining similarity between these two blocks that when extracted will make the differences more apparent.